packages/core/src/components/Dropdown/DropdownConstants.js

```diff
@@ -1,22 +0,0 @@
-export const defaultCustomStyles = baseStyles => baseStyles;
-
-export const ADD_AUTO_HEIGHT_COMPONENTS = ["container", "control", "valueContainer"];
-
-export const DROPDOWN_ID = "dropdown-menu-id";
-
-export const DROPDOWN_CHIP_COLORS = {
-  PRIMARY: "PRIMARY",
-  NEGATIVE: "NEGATIVE",
-  POSITIVE: "POSITIVE"
-};
-
-export const DROPDOWN_MENU_POSITION = {
-  ABSOLUTE: "absolute",
-  FIXED: "fixed"
-};
-
-export const DROPDOWN_MENU_PLACEMENT = {
-  TOP: "top",
-  BOTTOM: "bottom",
-  AUTO: "auto"
-};
```

Comment:
Since we are yet to support using types externally correctly we don't want to break existing consumers' usages when consuming these objects from the dist folder, so we'll have to keep them intact until next major version

Reply:
i understand, i will revert this change